addons/lunch/models/lunch_supplier.py

```diff
@@ -190,11 +190,11 @@ def create(self, vals_list):
 
     def write(self, values):
         for topping in values.get('topping_ids_2', []):
-            topping_values = topping[2]
+            topping_values = topping[2] if topping[0] not in [Command.UNLINK, Command.DELETE] else False
```

Comment:
I'm not sure I understand this condition, is there a reason why checking the length of `topping` is not enough ?
also can you please add a test for this fix ?
Thank you in advance :pray:

Reply:
Hello :wave: 
I added the test case.
Thanks!